addons/missionTest/functions/fnc_onMissionSave.sqf

```diff
@@ -86,56 +119,76 @@ private _checkMagazines = [];
                 } forEach (magazinesAmmoFull _unit);
                 if (_ammoCount == 0) then {
                     if ((toLower _weapon) in ["rhs_weap_rpg26", "rhs_weap_m136", "launch_nlaw_f"]) then {
-                        // TRACE_1("ignoring special AT reloads",_weapon);
+                        TRACE_1("ignoring special AT reloads",_weapon);
                     } else {
-                        WARNING_2("[%1]'s %2 has zero ammo", typeOf _unit, _weapon);
-                        _checkMagazines pushBackUnique (typeOf _unit);
+                        TRACE_2("has zero ammo",_typeOf,_weapon);
+                        _checkMagazines pushBackUnique format ["%1 has no ammo for %2", _typeOf, _weapon];
                     };
                 };
                 if (_weapon == (primaryWeapon _unit)) then {
-                    private _desiredAmmo = switch (true) do {
-                    case (((toLower typeOf _unit) find "marksman") > 0);
-                    case (((toLower typeOf _unit) find "spotter") > 0);
-                    case (((toLower typeOf _unit) find "sniper") > 0): { 10 };
-                    case (((toLower typeOf _unit) find "_ar") > 0);
-                    case (((toLower typeOf _unit) find "_mg") > 0): { 250 };
-                        default { 100 };
+                    private _desiredAmmo = call {
+                        // allow low ammo count for long range gunners
+                        if ((_typeOf find "marksman" > 0) || {_typeOf find "spotter" > 0} || {_typeOf find "sniper" > 0}) exitWith { 20 };
+                        // suggest hight ammo count for MGs
+                        if ((_typeOf find "_ar" > 0) || {_typeOf find "_mg" > 0} || {_typeOf find "_mmgg" > 0}) exitWith { 250 };
+                        // default rifleman case
+                        100
                     };
                     if (_ammoCount <= _desiredAmmo) then {
-                        WARNING_4("[%1]'s %2 has limited rifle ammo [%3/%4]", typeOf _unit, _weapon, _ammoCount, _desiredAmmo);
-                        _checkMagazines pushBackUnique (typeOf _unit);
+                        TRACE_4("limited rifle ammo",_typeOf,_weapon,_ammoCount,_desiredAmmo);
+                        _checkMagazines pushBackUnique format ["%1 only has %2", _typeOf, _ammoCount];
                     };
                 };
             };
         } forEach [primaryWeapon _x, secondaryWeapon _x, handgunWeapon _x];
+
+        // Check medical gear:
+        private _bandageCount = {(_x == "ACE_fieldDressing") || {_x == "ACE_packingBandage"} || {_x == "ACE_elasticBandage"}} count (items _unit);
+        if ((getNumber (configFile >> "CfgVehicles" >> _typeOf >> "attendant")) > 0) then {
```

Comment:
That uses ace settings, which should work in 3den, but might not if ace switches to cba settings